### ISSUE TITLE
Why am I running the result EER= 15.6369?

### ISSUE THREAD

**Issue by kingback2019** [josebeo2016/BTS-Encoder-ASVspoof#1]
Hello, I am trying to use the following command
~~~python
python main.py --config configs/model_config_RawNet_Trans_32concat.yaml --track=LA --loss=CCE --lr=0.0001 --batch_size=64 --database_path=asvspoof_2019_LA_data/ --protocols_path=asvspoof_2019_LA_data/
~~~

The model is trained and the following results are obtained on the training set:

~~~python
Now epoch: the 99 - train_loss: 0.0020150839754358666 - train_acc: 99.94 - valid_acc: 99.91,
~~~

But why can we only get `EER= 15.636914986` and `min-tDCF= 0.246558160` on the `ASVspoof2019_LA_eval` data set

Looking forward to your reply, thank you.

**Comment by josebeo2016** [comment]
Your experience is the same as mine. I also got the pooled EER and min-tDCF around 15 -17%. However, as I claimed in my paper, the BTS-E can only work well for TTS attacks. If you take a breakdown evaluation, you will see the EER is high for attacks from A17-19 (VC cases).

**Comment by kingback2019** [comment]
> 
Thanks again for your answer, after I removed the A17-A19 attack, the result seems normal.
